cve_bin_tool/cvedb.py

```diff
@@ -361,10 +356,7 @@ def openssl_convert(self, version):
             return version
 
         lastchar = version[len(version) - 1]
-        letters = {
-            letter: str(index)
-            for index, letter in enumerate("abcdefghijklmnopqrstuvwxyz")
-        }
+        letters = dict(zip(ascii_lowercase, range(26)))
```

Comment:
There was some reason I didn't do this using ascii_lowercase (I remember even looking it up), and I can't for the life of me remember what it was.  It's really bugging me now!

Reply:
I have look it up in cpython source code of [string](https://github.com/python/cpython/blob/master/Lib/string.py#L25) and it only contains "abcd...". so, there isn't any performance improvement while using ascii_lowercase. So, I think it's only for generalization.